src/Neo/SmartContract/Native/NativeContract.cs

```diff
@@ -140,11 +140,12 @@ protected NativeContract()
             // Reflection to get the methods
 
             List<ContractMethodMetadata> listMethods = [];
-            foreach (MemberInfo member in GetType().GetMembers(BindingFlags.Instance | BindingFlags.Static | BindingFlags.NonPublic | BindingFlags.Public))
+            foreach (var member in GetType().GetMembers(BindingFlags.Instance | BindingFlags.Static | BindingFlags.NonPublic | BindingFlags.Public))
             {
-                ContractMethodAttribute attribute = member.GetCustomAttribute<ContractMethodAttribute>();
-                if (attribute is null) continue;
-                listMethods.Add(new ContractMethodMetadata(member, attribute));
+                foreach (var attribute in member.GetCustomAttributes<ContractMethodAttribute>())
```

Comment:
Previously there was a check for `if (attribute is null)`, is it possible that attribute is nil with the current implementation? I think it's OK, but we'd better make sure.

Reply:
No, now can't be null